viz.yaml

```diff
@@ -294,74 +280,22 @@ publish:
     location: "js/topojson.v1.min.js"
     mimetype: application/javascript
   -
-    id: url_hashes
-    location: "js/url_hashes.js"
-    mimetype: application/javascript
-  # custom_projection.js must be before build_map.js for it to find the projection function.
-  -
-    id: custom_projection
-    location: "js/custom_projection.js"
+    id: all_minified
+    location: "js/all.min.js"
     mimetype: application/javascript
+    publish_args:
+      js_files: [
+        "js/url_hashes.js", "js/custom_projection.js", "js/styles.js", "js/buttons.js",
+        "js/resize.js", "js/map_utils.js", "js/states.js", "js/counties.js", "js/watermark.js", 
+        "js/circles.js", "js/mobile_ui.js", "js/modernizr_custom.js", "js/lazy_loading.js", 
+        "js/national_pie.js", "js/rank_states_utils.js", "js/scroll_tracking.js", "js/build_map_utils.js"
+      ]
+    publisher: combine_minify_js
+    scripts: [scripts/publish/combine_minify_js.R]
   -
     id: build_map
```

Comment:
I realize this is a small script and I understand the reason why this file is separate from the others, but should `build_map` be minified as well?

Reply:
Oh yeah, I can do that!